src/domain/event/eventInfo/EventInfo.tsx

```diff
@@ -253,7 +255,8 @@ const EventInfo: React.FC<Props> = ({ event, eventType }) => {
             ))}
             {externalLinks.map((externalLink, index) => {
               return (
-                !!externalLink.link && (
+                !!externalLink.link &&
```

Comment:
is `!!externalLink.link &&` redundant here?

Reply:
i am not sure can the link prop be empty or not.